### PR TITLE
onboarding stepper implementation

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

**Comment by DhaaraniCIT** [comment]
Screenshot 2023-02-06 at 1 38 19 PM

File: src/app/core/services/qbd/qbd-core/qbd-workspace.service.ts [hunk 1]
```diff
@@ -30,11 +30,11 @@ export class QbdWorkspaceService {
   }
 
   setOnboardingState(onboardingState: QBDOnboardingState): void {
-    return this.storageService.set('onboardingState', onboardingState);
+    return this.storageService.set('QBDonboardingState', onboardingState);
```
Comment (ashwin1111) [inline src/app/core/services/qbd/qbd-core/qbd-workspace.service.ts:33]: `QBDOnboardingState`

File: src/app/core/services/qbd/qbd-core/qbd-workspace.service.ts [hunk 2]
```diff
@@ -30,11 +30,11 @@ export class QbdWorkspaceService {
...
   }
 
   getOnboardingState(): QBDOnboardingState {
-    const onboardingState = this.storageService.get('onboardingState');
-    return onboardingState ? onboardingState : QBDOnboardingState.CONNECTION;
+    const onboardingState = this.storageService.get('QBDonboardingState');
```
Comment (ashwin1111) [inline src/app/core/services/qbd/qbd-core/qbd-workspace.service.ts:37]: `QBDOnboardingState`

File: src/app/shared/components/qbd/helper/onboarding-stepper/onboarding-stepper.component.html [hunk 3]
```diff
@@ -1 +1,36 @@
...
+                        <img *ngIf="onboardingStep.active" src='{{ "assets/icons/stepper/" + onboardingStep.icon + ".svg" }}' class="icon" >
+                        <img *ngIf="!onboardingStep.active && onboardingStep.completed" src="assets/icons/general/tick.svg" class="icon">
+                        <img *ngIf="!onboardingStep.active && !onboardingStep.completed" src='{{ "assets/icons/stepper/" + onboardingStep.icon + "-inactive.svg" }}' class="icon">
+                    </div>
+                    <div *ngIf="onboardingStep.number !== 4" >
+                        <div *ngIf="!onboardingStep.completed">
+                            <img src="../../../../../../assets/icons/line-inactive.svg" alt="" srcset="">
+                        </div>
+                        <div *ngIf="onboardingStep.completed">
+                            <img src="../../../../../../assets/icons/line.svg" alt="" srcset="">
```
Comment (ashwin1111) [inline src/app/shared/components/qbd/helper/onboarding-stepper/onboarding-stepper.component.html:22]: can we use `svg-icon-sprite` please?

File: src/app/shared/components/qbd/helper/onboarding-stepper/onboarding-stepper.component.scss [hunk 4]
```diff
@@ -0,0 +1,15 @@
+  .step {
```
Comment (ashwin1111) [inline src/app/shared/components/qbd/helper/onboarding-stepper/onboarding-stepper.component.scss:1]: indent is off, it should be aligned close to margin without any space

**Review by ashwin1111 — APPROVED** [review]

**Pull request merged**